test/drenv/envfile.py

```diff
@@ -2,9 +2,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import copy
+import platform
 
 import yaml
 
+VM = "drenv-vm"
+CONTAINER = "drenv-container"
+SHARED_NETWORK = "drenv-network-shared"
+
+
+def platform_defaults():
+    # Let minikube defaults and errors apply if unknown platforms
+    d = {VM: "", CONTAINER: "", SHARED_NETWORK: ""}
+
+    if platform.platform() == "linux":
+        d[VM] = "kvm2"
+        d[CONTAINER] = "podman"
+        d[SHARED_NETWORK] = "default"
+    elif platform.platform() == "darwin":
+        d[VM] = "hyperkit"
+        d[CONTAINER] = "podman"
+        d[SHARED_NETWORK] = ""
+
+    return d
```

Comment:
I think this should be `platform.system().lower()` instead of `platform.platform()` 

```
In [1]: import platform

In [2]: platform.platform()
Out[2]: 'Darwin-22.3.0-x86_64-i386-64bit'

In [3]: platform.system().lower()
Out[3]: 'darwin'
```